store/modules/actions/authCore.js

```diff
@@ -127,10 +127,8 @@ export async function fetchAuthCoreCosmosWallet({ state }) {
 }
 
 export async function prepareAuthCoreCosmosTxSigner({ state }) {
-  let cosmosProvider;
   if (!state.cosmosProvider) throw new Error('COSMOS_WALLET_NOT_INITED');
-  ({ cosmosProvider } = state); // eslint-disable-line prefer-const
-
+  const { cosmosProvider } = state;
   if (cosmosProvider) {
```

Comment:
Please remove the if part for `if (cosmosProvider) {` since `if (!state.cosmosProvider)` already guard this case, just use `cosmosProvider` directly